### PR TITLE
[READY]Adds a Tutorial area for three jobs

### PR DESCRIPTION
**Pull request by Iamgoofball**
This adds the Tutorial Z-level, which you can access via an observer verb.

This is a Z-level devoted to being an area new players can use to learn more complex and freeform game mechanics, such as Chemistry, or learn the math behind Telescience.

It has:
Toxins Lab, with Canister Generator.

Chemistry Lab, with Assembly Generator.

Telescience lab, with teleporters with maxed out power capacity and crystal amounts so you can learn to telescience unrestricted. It, however, can only teleport around on the 7th z-level.

A few random spots out in the z-level for Telescience to target and attempt to reach, that have GPSes, to challenge your mathematics skill to grab all of them.

This also adds the Enter Thunderdome verb. This verb will spawn you on the thunderdome as a human so that you can have a fight with other ghosts.

PS. DME change is required, I ticked toxinsdispenser.dm

### CODE REVIEW

File: code/modules/mob/dead/observer/observer.dm [hunk 1]
```diff
@@ -150,6 +150,17 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
...
 
+/mob/dead/observer/verb/gototutorial()
+	set category = "Ghost"
+	set name = "Enter Training Area"
+	if(!istype(usr, /mob/dead/observer))
+		usr << "Not when you're not dead!"
+		return
+	var/mob/living/carbon/human/dummy/A
+	A.loc = pick(tutorialstart)
+	A.name = "TESTING HUMANOID CYBORG MODEL NO.[pick("1","2","3","4","5","6","7","8","9","0")][pick("1","2","3","4","5","6","7","8","9","0")][pick("1","2","3","4","5","6","7","8","9","0")][pick("1","2","3","4","5","6","7","8","9","0")][pick("1","2","3","4","5","6","7","8","9","0")][pick("1","2","3","4","5","6","7","8","9","0")][pick("1","2","3","4","5","6","7","8","9","0")]"
```
Comment (Atskadan) [inline code/modules/mob/dead/observer/observer.dm]: is there not a better way to do this

Comment (Iamgoofball) [inline code/modules/mob/dead/observer/observer.dm]: I can't think of any.

Comment (RemieRichards) [inline code/modules/mob/dead/observer/observer.dm]: Replace pick(Bollocks) with rand(1,999) you pleb, all 3 picks can be replaced by just one rand(1,999)

Comment (Iamgoofball) [inline code/modules/mob/dead/observer/observer.dm]: Will do.

**Comment by Iamgoofball** [comment]
Alright, this PR should be ready for review finally.

**Comment by Neerti** [comment]
Can you make it a second thunderdome since admins might want to do stuff in the centcom one?

**Comment by Iamgoofball** [comment]
Sure, can do.

**Comment by RemieRichards** [comment]
No offense @Iamgoofball but can you hurry this one up? It has Z level changes so It's holding off my work on that.

**Comment by GunHog** [comment]
It is marked "[READY]".

**Comment by Iamgoofball** [comment]
@GunHog I think he means the thunderdome thing

I'm doing that right now

**Comment by Iamgoofball** [comment]
will re-open on new branch because fukken merge issues

**Comment by RemieRichards** [comment]
right

**Pull request closed without merging**